### ISSUE TITLE
how to resolve the error

### ISSUE THREAD

**Issue by liwei00wade** [MetaMask/metamask-extension#5914]
ubuntu 
npm i    // ok 
npm start   // error 

: Error: Cannot find module '@babel/runtime/core-js/map' from '/home/wade/goWorkSpace/src/wade/metamask-extension/node_modules/@material-ui/core/styles'
Error: Cannot find module '@babel/runtime/core-js/object/get-prototype-of' from '/home/wade/goWorkSpace/src/wade/metamask-extension/node_modules/@material-ui/core/styles'
Error: Cannot find module '@babel/runtime/core-js/object/keys' from '/home/wade/goWorkSpace/src/wade/metamask-extension/node_modules/@material-ui/core/styles'
Error: Cannot find module '@babel/runtime/core-js/object/get-prototype-of' from '/home/wade/goWorkSpace/src/wade/metamask-extension/node_modules/@material-ui/core/styles'
Error: Cannot find module '@babel/runtime/core-js/object/get-prototype-of' from '/home/wade/goWorkSpace/src/wade/metamask-extension/node_modules/@material-ui/core/styles'
Error: Cannot find module '@babel/runtime/core-js/number/min-safe-integer' from '/home/wade/goWorkSpace/src/wade/metamask-extension/node_modules/@material-ui/core/styles'
........

**Comment by tmashuang** [comment]
Whats the npm/node version?
Possibly need to `npm run dist` before start.

**Comment by liwei00wade** [comment]
wade@wade-work:~/$ node -v
v8.11.3
wade@wade-work:~/$ npm -v
5.6.0
wade@wade-work:~/goWorkSpace/src/github.com/MetaMask/metamask-extension$

**Comment by tmashuang** [comment]
Please read https://github.com/metamask/metamask-extension#building-locally

**Comment by liwei00wade** [comment]
That is great,
Thanks a lot.

**Comment by tmashuang** [comment]
TLDR; Install Node.js version 8 and the latest available npm@6, specifically node v8.13

**Comment by liwei00wade** [comment]
Thank you again,
I have got it.
